vm/src/frame.rs

```diff
@@ -349,7 +349,16 @@ impl ExecutingFrame<'_> {
         // Execute until return or exception:
         let instrs = &self.code.instructions;
         let mut arg_state = bytecode::OpArgState::default();
+        let mut gc_count = 0;
         loop {
+            gc_count += 1;
+            if gc_count > 1000 {
+                #[cfg(feature = "gc_bacon")]
+                {
+                    crate::object::try_gc();
+                }
+                gc_count = 0;
+            }
```

Comment:
> Idk well about gc, so I'd like to learn about the reasoning. is this the preferred place to put try_gc? and how the number 1000 is decided? isn't it a potential cause the reason gc is too slow because it is too small?

My bad, I should mimic CPython(https://github.com/python/cpython/blob/7703def37e4fa7d25c3d23756de8f527daa4e165/Python/ceval.c#L833), here in handlePending it does necessary check whether to gc or not, I should rewrite this part